cli/functionary/package.py

```diff
@@ -17,15 +17,16 @@ def create_languages() -> list[str]:
 
 
 def generateYaml(output_dir: str, name: str, language: str):
-    metadata = {
-        "name": name,
-        "version": "1.0",
-        "x-language": language,
-    }
-
-    path = pathlib.Path(output_dir).resolve() / name / f"{name}.yaml"
-    with path.open(mode="w"):
-        path.write_text(yaml.dump(metadata))
+    path = pathlib.Path(output_dir).resolve() / name / "package.yaml"
```

Comment:
Let's use descriptive names for the paths.  `package_path` and `template_path` rather than `path` and `path2`.